spark/v3.4/spark/src/main/java/org/apache/iceberg/spark/source/SparkScanBuilder.java

```diff
@@ -96,6 +97,7 @@ public class SparkScanBuilder
   private boolean caseSensitive;
   private List<Expression> filterExpressions = null;
   private Filter[] pushedFilters = NO_FILTERS;
+  private final InMemoryReadMetricReporter metricsReporter;
```

Comment:
Minor: This should go to the block with final variables above, you may init it in in the definition (up to you).

```
private final SparkReadConf readConf;
private final List<String> metaColumns = Lists.newArrayList();
private final InMemoryMetricsReporter metricsReporter = new InMemoryMetricsReporter();
```

Reply:
I think this still applies.